admin/admin.go

```diff
@@ -35,6 +36,7 @@ type Service struct {
 	issuer           *auth.Issuer
 	VersionNumber    string
 	metricsProjectID string
+	AutoScalerCron   string
```

Comment:
nit: Change to `AutoscalerCron` (two words, not three)

Reply:
Good Catch!